File: algorithm-programmers-stackqueue-feature-development/src/test/java/com/currenjin/SolutionTest.java
package com.currenjin;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class SolutionTest {
    @Test
    void sample1() {
        int[] actual = new Solution().solution(new int[]{93, 30, 55}, new int[]{1, 30, 5});
        assertArrayEquals(new int[]{2, 1}, actual);
    }

    @Test
    void sample2() {
        int[] actual = new Solution().solution(
                new int[]{95, 90, 99, 99, 80, 99},
                new int[]{1, 1, 1, 1, 1, 1}
        );
        assertArrayEquals(new int[]{1, 3, 2}, actual);
    }
}

File: algorithm-programmers-stackqueue-feature-development/src/test/kotlin/com/currenjin/SolutionTest.kt
package com.currenjin

import org.junit.jupiter.api.Assertions.assertArrayEquals
import org.junit.jupiter.api.Test

class SolutionTest {
    @Test
    fun sample1() {
        val actual = Solution().solution(
            intArrayOf(93, 30, 55),
            intArrayOf(1, 30, 5)
        )
        assertArrayEquals(intArrayOf(2, 1), actual)
    }

    @Test
    fun sample2() {
        val actual = Solution().solution(
            intArrayOf(95, 90, 99, 99, 80, 99),
            intArrayOf(1, 1, 1, 1, 1, 1)
        )
        assertArrayEquals(intArrayOf(1, 3, 2), actual)
    }
}
